src/main/java/uk/gov/indexer/ctserver/Audit.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.indexer.ctserver;
+
+import java.util.List;
+
+public class Audit {
```

Comment:
I'm not sure Audit is the right name for this -- based on the [endpoint name & docs](http://tools.ietf.org/html/rfc6962#section-4.5), maybe "Proof" or "AuditProof"?